src/main/java/org/codehaus/mojo/versions/api/ArtifactVersions.java

```diff
@@ -187,12 +187,7 @@ public VersionComparator getVersionComparator()
      */
     public String toString()
     {
-        final StringBuilder sb = new StringBuilder();
-        sb.append( "ArtifactVersions" );
-        sb.append( "{artifact=" ).append( artifact );
-        sb.append( ", versions=" ).append( versions );
-        sb.append( ", versionComparator=" ).append( versionComparator );
-        sb.append( '}' );
-        return sb.toString();
+        return "ArtifactVersions" + "{artifact=" + artifact + ", versions=" + versions + ", versionComparator="
+            + versionComparator + '}';
```

Comment:
Consider a `StringJoiner` here, which is more efficient and easier to extend.

Reply:
c'mon it;'s just a toString() method